src/colr.c

```diff
@@ -108,7 +108,7 @@ struct avifMatrixCoefficientsTable
     const float kb;
 };
 
-// https://www.itu.int/rec/T-REC-H.273-201612-I/en
+// https://www.itu.int/rec/T-REC-H.273-201612-S
```

Comment:
I am curious: in France is this URL redirected to https://www.itu.int/rec/T-REC-H.273-201612-S/fr?

Reply:
yes it does (from my personal account)